src/dynarec/rv64/dynarec_rv64_d9.c

```diff
@@ -186,7 +192,6 @@ uintptr_t dynarec64_D9(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
             MARK3;
             // Extract signa & Update SW
             SRLI(x1, x2, 63);
-            ANDI(x4, x4, ~(1<<9));
```

Comment:
the the bit 9 needs to be cleared beforre the OR

Reply:
or is it already 0 no mater what?

Reply:
That bit in x4 are always zero because it’ constructed manually above.